tests/engine/eval/filter.rs

```diff
@@ -43,3 +43,24 @@ fn test_filter_chain() {
     test_eval_eq!("`a` | lower | upper", json!("A"));
     test_eval_eq!("`A` | lower | upper", json!("A"));
 }
+
+#[test]
+fn test_custom_filter() {
+    let cf = |value: &Value| {
+        if value.is_string() {
+            Ok(Value::String(value.as_str().unwrap().replace("a", "b")))
+        } else {
+            Err("no string, no fun".into())
+        }
+    };
+
+    let mut engine = Engine::default();
+    engine.register_filter("atob", cf);
+    let ctx = Context::default();
```

Comment:
from the point of view of the user - in this case - do they need to know about `context` ? Would it be possible/make sense to have a interface where they just do `eval` ?

Reply:
Yes, they must know about it. `context` contains DRY json (data) and will contain other things in the future.